scripts/github-actions/example-ci.yml

```diff
@@ -44,5 +44,12 @@ jobs:
         run: ${{ matrix.runs.history }}
       - name: Test ${{ matrix.name }}
         run: ${{ matrix.runs.test }}
-      - name: '"docker images"'
+      - name: 'List all built docker images'
         run: ${{ matrix.runs.images }}
+      - name: 'Scan image'
+        uses: aquasecurity/trivy-action@master
+        with:
+          image-ref: '${{ matrix.runs.getfirsttag }}'
```

Comment:
I think `matrix.meta.entries[0].tags[0]` is probably more precisely what you're looking for here (and already available :eyes:).

One thing to be aware of is that if you have any inter-dependent tags, this will only catch the first tag of the first entry (but it wouldn't be too complicated to take `matrix.meta.entries` and do something more complex with it -- I usually do so with `jq`).

Reply:
Thanks! I can use `matrix.meta.entries[0].tags[0]` instead.